### PR TITLE
use input instead of attribute

### PR DESCRIPTION
**Pull request by micheelengronne**
In the last versions of Inspec and cinc-auditor, attribute is deprecated and input should be used.

https://docs.chef.io/workstation/cookstyle/inspec_deprecations_attributehelper/

### CODE REVIEW

**Comment by micheelengronne** [comment]
@chris-rock @schurzi do you agree ?

**Review by schurzi — APPROVED** [review]
LGTM

File: Rakefile [hunk 1]
```diff
@@ -20,23 +19,30 @@ task default: [:lint, 'test:check']
...
+    profile = Inspec::Profile.for_target('.', backend: Inspec::Backend.create(Inspec::Config.mock))
+    pp profile.check
   end
 end
 
-# Automatically generate a changelog for this project. Only loaded if
-# the necessary gem is installed. By default its picking up the version from
-# inspec.yml. You can override that behavior with `rake changelog to=1.2.0`
-begin
+task :changelog do
```
Comment (schurzi) [inline Rakefile:29]: the Changelog is generated by a Github action (https://github.com/dev-sec/cis-dil-benchmark/blob/master/.github/workflows/release.yml), there is no need to keep this here

Comment (micheelengronne) [reply Rakefile:29]: Ok, I'll remove it.

**Review by chris-rock — APPROVED** [review]

**Pull request merged**